mobile-sdk/merchant-backend.md

```diff
@@ -0,0 +1,476 @@
+| `token`               | `string` | A session token used to initiate Checkout UI.                                                                                                     |
+| `operations`          | `array`  | The array of operation objects to choose from, described in detail in the table below.                                                            |
+| └➔&nbsp;`rel`         | `string` | The relational name of the operation, used as a programmatic identifier to find the correct operation given the current state of the application. |
+| └➔&nbsp;`method`      | `string` | The HTTP method to use when performing the operation.                                                                                             |
+| └➔&nbsp;`contentType` | `string` | The HTTP content type of the target URI. Indicates what sort of resource is to be found at the URI, how it is expected to be used and behave.     |
+| └➔&nbsp;`href`        | `string` | The target URI of the operation.                                                                                                                  |
+
+## Payment Orders Endpoint
+
+The `paymentorders` endpoint is used to create a new Payment Order. It is specified as a transparent wrapper around the corresponding [Swedbank Pay API][create-payment-order]. However, it is to be expected that your backend will need to process the payment order both before making the Swedbank Pay API call, and after receiving the response from Swedbank Pay. The sample implementations validate the input, then create an internal unique identitifer for the payment order, and set that as `paymentorder.payeeInfo.payeeReference`, before making the Swedbank Pay call. After receiving the response, the backend stores the `id` of the Payment Order for future use, and forwards the response to the SDK.
+
+A production implementation should validate the payment order also from a business logic perspective. This is, naturally, outside the scope of the SDK, as is any other processing you may wish to perform with the payment order. The SDK expects the same form of response as returned from the Swedbank Pay API. Specifically, the response must contain the `view-paymentorder` operation.
+
+{:.code-header}
+**Request**
+
+```http
+POST /swedbank-pay-mobile/paymentorders HTTP/1.1
+Host: example.com
+Your-Api-Key: secretish
+Content-Type: application/json
+
+{
+    "paymentorder": { "①": "" }
+}
+```
+
+{:.table .table-striped}
+|     Required     | Field          | Type     | Description                                                  |
+| :--------------: | :------------- | :------- | :----------------------------------------------------------- |
+| {% icon check %} | `paymentorder` | `object` | The payment order to create                                  |
+
+* ① The contents of `paymentorder` are omitted here. See [Checkout Documentation](create-payment-order) for details.
```

Comment:
Codacy found an issue: [[no-reference-like-url] Did you mean to use `[create-payment-order]` instead of `(create-payment-order)`, a reference?](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5758572)